.github/workflows/deb-packaging.yml

```diff
@@ -0,0 +1,271 @@
+name: "Ubuntu packaging"
```

Comment:
This file is very heavy with logic. Would it make sense to move more of it to a .sh script?

Reply:
Moved some parts to a `linux/scripts/deb-packaging.sh` script.